src/site/markdown/server/manage-webadmin.md

```diff
@@ -2496,6 +2497,45 @@ Response codes:
 
 Not implemented yet.
 
+## Deleted Messages Vault
+
+Deleted messages from an user will be stored in a restriction place where only delegated administrator can access.
+This place is called the Vault. Messages in the Vault will be deleted permanently by a configured retention policy.
+
+ - [Restore Deleted Messages](#Restore_deleted_messages)
+
+### Restore Deleted Messages
+
+An user can request to the administrator to restore his messages had been deleted and now at the Vault.
+To restore them, the administrator perform a http request:
+
+```
+curl -XPOST http://ip:port/deletedMessages/user/userToRestore@domain.ext
+```
+
+**All** messages from the user passed in the path param will be moved from the Vault to his new mailbox.
```

Comment:
I may miss something but I'm pretty sure we don't want to implement a "restore all" feature.

Reply:
I want to separate restore route into two PR, the first one is restore all, the second one is replacing restore all by restore by query. Because put them together in one PR makes it become complicated to review and comes with a slow progress.

Reply:
It doesn't make sense to document something that is not the feature (even if the code does that at one point) but I don't mind.